documentation/api/development.md

```diff
@@ -11,17 +11,28 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
 1. Install the version of Python specified in [pyproject.toml](../../api/pyproject.toml)
    [pyenv](https://github.com/pyenv/pyenv#installation) is one popular option for installing Python,
    or [asdf](https://asdf-vm.com/).
-
-2. After installing and activating the right version of Python, install
+   - If using pyenv run `pyenv local <version>` to ensure that version will be used in subsequent steps
+2. Ensure that `python -V` and `python3 -V` are picking up that version.
+   - If not, run `pyenv init -` and/or restart your shell to ensure it was run automatically
+3. After installing and activating the right version of Python, install
    [poetry](https://python-poetry.org/docs/#installation) and follow the instructions to add poetry to your path if necessary.
 
    ```bash
    curl -sSL https://install.python-poetry.org | python3 -
    ```
 
-3. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
+4. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
+
+5. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+## Database setup: Run Migrations/Seeds
+
+1. If you haven't done local development before you'll need to execute the migrations and seed the DB with data using the steps in [database-local-usage.md](database/database-local-usage.md)
+
+## OpenSearch setup
 
-4. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+1. Run `make init-opensearch` setup the OpenSearch Container
+2. Run `make populate-search-opportunities` to push data previously seeded in the DB into the search index
```

Comment:
Might be worth calling out the `make volume-recreate` command as well. In drops the docker volumes and can delete any data you already have locally. Useful if you want to replace your DB data or your DB migrations are in a weird state.

Reply:
Added reference and cleaned up one later in the file to distinguish this from "clean-volumes"